docs/source/cuda_ann_benchmarks.md

```diff
@@ -60,6 +60,143 @@ There are 4 general steps to running the benchmarks:
 3. Search Using Built Index
 4. Evaluate Result
```

Comment:
Can we move all the build content to a new markdown file and link to it here? Eventually, I'd like to create conda packages from the benchark executables so users can just install them and go to town. I want to try and make this as easy to use as possible.